nodi_backend/urls.py

```diff
@@ -18,5 +18,6 @@
 from django.views.generic import TemplateView
 
 urlpatterns = [
-    re_path(r'^(?:.*)/?$', TemplateView.as_view(template_name='index.html'))
+    re_path(r'^(?:.*)/?$', TemplateView.as_view(template_name='index.html')),
+    path('api/datasets/', views.LeadListCreate.as_view()),
 ]
```

Comment:
You'll need to put the `api/datasets` path above the regex index.html catchall, otherwise the api won't get hit at all.